srcpkgs/ascii-image-converter/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'ascii-image-converter'
+pkgname='ascii-image-converter'
+short_desc="Tool to convert images into ascii art and braille art"
+maintainer='Nadeŭka <nadevko@riseup.net>'
+license='Apache-2.0'
+homepage='https://github.com/TheZoraiz/ascii-image-converter'
+revision='1'
+
+version='1.11.0'
+checksum='17f87445971fdb491c6ca9d4beed67044a330339b47775adffb8e3cb898ec15d'
+
+distfiles="https://github.com/TheZoraiz/ascii-image-converter/releases/download/v$version/ascii-image-converter_Linux_amd64_64bit.tar.gz>v$version.tar.gz"
+nopie='/usr/bin/ascii-image-converter'
+
+do_extract() {
+        bsdtar --strip-components 1 -xf $XBPS_SRCDISTDIR/${wrksrc##*/}/v$version.tar.gz
+}
```

Comment:
should not be necessary, xbps-src knows how to unpack tar.gz and zip files itself